models/job.go

```diff
@@ -0,0 +1,112 @@
+// mystack-controller api
+// https://github.com/topfreegames/mystack-controller
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2017 Top Free Games <backend@tfgco.com>
+
+package models
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/topfreegames/mystack-controller/errors"
+	"text/template"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/pkg/apis/batch"
+	"k8s.io/client-go/pkg/apis/batch/v1"
+)
+
+const jobYaml = `
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{.Name}}
+  namespace: {{.Namespace}}
+  labels:
+    mystack/routable: "true"
```

Comment:
I use it in some tests to ensure that the job is running. Does it mess with mystack-logger? I can change it to check in another way.